source/2019-05-14-ember-3-10-released.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Ember 3.10 Released
+author: Kenneth Larsen, Jessica Jordan
+tags: Releases, 2019, 3, 3.10
+responsive: true
+---
+
+Today the Ember project is releasing version 3.10 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.11 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.10
+Ember.js 3.10 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There are four (4) new features, one (1) deprecations, and fifteen (15) bugfixes in this version.
```

Comment:
There are 17 bugfixes in the CHANGELOG, but this is hard to count; I think there was chatter about dropping this enumeration? 

It's hard to count because there are many bugfixes that are deemed not worthy of putting in the CHANGELOG, or that get backported as well and so are removed from the CHANGELOG for the particular version

Reply:
Yeah, I agree it is a bit weird. I remember that some people felt strongly about keeping the count so I just kept it until we agree on something